apps/trading-e2e/src/support/step_definitions/vega-wallet.step.js

```diff
@@ -0,0 +1,42 @@
+import { When } from 'cypress-cucumber-preprocessor/steps';
+import VegaWallet from '../vega-wallet';
+
+const vegaWallet = new VegaWallet();
+
+beforeEach(() => {
+  cy.intercept('POST', 'http://localhost:1789/api/v1/command/sync', {
+    body: {
+      txHash: 'test-tx-hash',
+      tx: {
+        input_data:
+          'CPe6vpiqsPqxDBDC1w7KPkoKQGE4Y2M0NjUwMjhiMGY4OTM4YTYzZTEzNDViYzM2ODc3ZWRmODg4MjNmOWU0ZmI4ZDRlN2VkMmFlMzAwNzA3ZTMYASABKAM4Ag==',
+        signature: {
+          // sig value needs to be 'real' so sigToId function doesn't error out
+          value:
+            'd86138bba739bbc1069b3dc975d20b3a1517c2b9bdd401c70eeb1a0ecbc502ec268cf3129824841178b8b506b0b7d650c76644dbd96f524a6cb2158fb7121800',
+          algo: 'vega/ed25519',
+          version: 1,
+        },
+        From: {
+          PubKey:
+            // TODO: Use same pubkey as the test wallet
```

Comment:
chore (non-blocking): todo? Issue, resolve or remove please :pray:

Reply:
Possibly the best solution is to move this to the env file so people can chnage it for local ect.